src/custom/pages/Swap/SwapMod.tsx

```diff
@@ -431,13 +432,12 @@ export default function Swap({
 
   // const priceImpactTooHigh = priceImpactSeverity > 3 && !isExpertMode
 
-  const [exactInLabel, exactOutLabel] = useMemo(
-    () => [
-      independentField === Field.OUTPUT && !showWrap && trade ? <Trans>From (incl. fee)</Trans> : null,
-      independentField === Field.INPUT && !showWrap && trade ? <Trans>Receive (incl. fee)</Trans> : null,
```

Comment:
why is showWrap removed here?

Reply:
it wasn't I undid the change here, and i left the useMemo with a plain old function with return instead of arrow function. I will use the arrow fn in the top of the waterfall